spec/features/invite_existing_user_spec.rb

```diff
@@ -65,34 +65,9 @@
         click_on "Send invitation email"
       end
 
-      it "does not send an invitation" do
-        expect(InviteUseCaseSpy.invite_count).to eq(0)
-        expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
-      end
-
-      context 'when user uses the confirmation link after invitation attempt' do
```

Comment:
I think a user should still be able to click on the confirmation link, so this context should remain

Reply:
understand now that this relates to the user being recreated. While it's not an ideal user journey, we don't expect people to use previous confirmation emails when a new one has just been sent to them

Reply:
The user record(s) get deleted when they are sent an invite so the previous confirmation link(s) would have been lost.